german/the-black-hack.md

```diff
@@ -46,9 +46,9 @@ Rüstung bietet Schutz, indem sie allen erlittenen Schaden reduziert. Jedes Stü
 
 |     Typ        | Punkte |
 |:--------------:|:------:|
-|  Festes Wams   |   2    |
-|  Lederrüstung  |   4    |
-|   Kettenhemd   |   6    |
+| Gambeson       |   2    |
```

Comment:
Hieß es unter anderem auch, wobei ich das einfach für falsch halte, da ein "Waffenrock" bzw. "Waffenkleid" das bunte "Teamtrikot" ist, was man über einer Metallrüstung trägt.

Bei der "Tuchrüstung" bin ich mir relativ sicher, dass das eine DSA-Wortschöpfung ist, die sonst nirgends existiert. Vor allem, weil das, was bei DSA damit beschrieben ist ein "Linothorax" ist, eine griechische Rüstung aus übereinander geklebten Leinenstücken, die angeblich (es gibt nur Text- und Bildquellen, keine verwertbaren Rüstungsfunde) ähnlich wie eine Kevlarweste funktioniert haben soll. Das ist das Ding, was Alexander der Große auf dem bekannten Mosaik trägt.

"Stoffrüstung" hat sich bei World of Warcraft und ähnlichen Computerspielen als Oberbegriff für Roben, Tuniken, Hüte und andere textile Ausrüstung durchgesetzt, die meistens vor allem Zauberklassen zur Verfügung steht. Damit kommt es mir für den unbedarften Leser ein wenig irreführend vor, da bei TBH der *Conjurer* diese Rüstung nicht ohne Behinderung tragen kann. Der Punkt ist, dass es sich schon um eine explizite Rüstung handelt und nicht einfach nur um Kleidung, die bei Nichtgerüsteten die entsprechenden Ausrüstungsslots füllt.

"Polsterrüstung" wäre wahrscheinlich technisch richtig (die Schutzwirkung besteht vor allem daraus, dass das Füllmaterial kinetische Energie von Hieben gut aufnimmt), aber da stellt man sich vielleicht auch einen plüschigen Sofakrieger vor?

"Wams" kann man zumindest theoretisch im Duden finden (wobei das natürlich auch ein antiquierter Begriff und den Lesern wahrscheinlich eher als Kürzel für "Welt am Sonntag" bekannt ist). Passt auch nicht wirklich gut zu dem ansonsten einsteigerfreundlichen Vokabular des Spiels.

Von der Idee her würde auch "Schutzweste" funktionieren, aber das hat für mich eindeutig moderne Anklänge. Ebenso "Schutzkleidung" oder "Textilrüstung".

Hmm.